cloudbuild.yaml

```diff
@@ -0,0 +1,60 @@
+steps:
+  - id: 'fetch-cache'
+    name: gcr.io/cloud-builders/gcloud
+    entrypoint: sh
+    args:
+      - -c
+      - |
+        (
+          set -e
```

Comment:
Can you please document the reasoning and logic?

Reply:
Sure, also probably worth describing the needed permissions and setup for `gcs` to store this, as I'm currently using `hoprd-cache` which I created myself manually. Not sure what would be the best process here?